libraries/chain/block_log.cpp

```diff
@@ -746,6 +779,23 @@ namespace eosio { namespace chain {
       my->head.reset();
    }
 
+
+   std::shared_ptr<std::vector<char>> detail::block_log_impl::read_block_stream_by_num(uint32_t block_num) {
```

Comment:
`read_block_buffer_by_num` I think would be a better name.

Reply:
This needs to take an argument for if packed `signed_block` or `signed_block_v0` should be returned.

Reply:
> read_block_buffer_by_num I think would be a better name.
Fixed

Reply:
> This needs to take an argument for if packed signed_block or signed_block_v0 should be returned.

Fixed.